tutorial/practicals/nemo/3_nemo_openmp/README.md

```diff
@@ -3,12 +3,11 @@
 This tutorial follows on from Tutorials 1 and 2
 (../1_nemo_psyir/README.md and ../2_nemo_profiling/README.md) and
 assumes that you are comfortable with the topics covered there. It
-uses PSyclone to parallelise the tracer-advection mini-app by adding
-OpenMP directives.
+uses PSyclone to parallelise the tracer-advection mini-app to make
+use of a multi-core CPU by adding appropriate OpenMP directives.
 
 You can find information on the various transformations supported by
-PSyclone in the [User Guide]
-(https://psyclone.readthedocs.io/en/stable/transformations.html).
+PSyclone in the [User Guide](https://psyclone.readthedocs.io/en/stable/transformations.html).
```

Comment:
This still points to outdated stable docs

Reply:
Also the same links on Tutorial 4

Reply:
Oops!